install/install.sh

```diff
@@ -27,7 +27,7 @@ echo ""
 echo "* Updating package list"
 apt-get update >> "$install_log" 2>&1
 echo "* Installing nginx, python, pip, kvm, libvirt and virt-manager"
-apt-get -q -y install nginx python python-dev python3-libvirt libxslt1-dev libxml2-dev python-pip qemu-kvm libvirt-bin virt-manager mediainfo >> "$install_log" 2>&1
+apt-get -q -y install nginx python3 python-is-python3 python3-libvirt libxslt1-dev python3-libxml2 python3-pip qemu-system-x86 libvirt-daemon-system libvirt-clients virt-manager mediainfo >> "$install_log" 2>&1
```

Comment:
Do we have tests to make sure that the dependency changes are not breaking in nature?

Reply:
I have installed the platform twice using this install.sh now. 
Also most of the dependencies replaced were not being installed and gave some error in the Platform Installation logs either because they were named or discontinued.

Reply:
And how many of these were caused by using a different Linux flavour than the Debian one we installed on? I'm just wondering...

Reply:
> And how many of these were caused by using a different Linux flavour than the Debian one we installed on? I'm just wondering...

I used Ubuntu 21.10 and pop_os 21.10